posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -233,6 +233,16 @@ def to_path_scroll_bounce_query(self) -> ast.SelectQuery:
                 },
             )
         assert isinstance(query, ast.SelectQuery)
+
+        # Compute query order based on the columns we're selecting
+        # Use our default order if we don't have any columns to order by
+        columns = [select.alias for select in query.select if isinstance(select, ast.Alias)]
```

Comment:
I think it's be good to handle all of the order by logic in one place, so either here or in _order_by

You're already passing columns into _order_by so it should have all the information to do.

Reply:
Not strongly! I do think that we probably want the sort order to have more to it than [sort, breakdown_value] though.

E.g. I would want
* if I sort by views, it should be [views, visitors, breakdown_value]
* if I sort by visitor, it should be [visitors, views, breakdown_value]

The way I describe seemed to me to be the easiest way to achieve this but there's many other ways too :)